views/template/head.ejs

```diff
@@ -2,22 +2,27 @@
 <script async src="https://www.googletagmanager.com/gtag/js?id=UA-127071715-1"></script>
 <script>
     window.dataLayer = window.dataLayer || [];
-    function gtag(){dataLayer.push(arguments);}
+    function gtag() { dataLayer.push(arguments); }
     gtag('js', new Date());
 
     gtag('config', 'UA-127071715-1');
 </script>
 
 <meta charset="utf-8">
 <title>Coding for Good</title>
-<link rel="shortcut icon" type="image/x-icon" href="/favicon.ico"/>
-<link rel="stylesheet" type="text/css" href="/css/bootstrap.min.css">
-<link rel="stylesheet" type="text/css" href="/css/style.css"/>
-<link rel="stylesheet" type="text/css" href="/css/neutra/stylesheet.css"/>
-<link rel="stylesheet" type="text/css" href="/css/jquery-editable-select.min.css"/>
+<link rel="shortcut icon" type="image/x-icon" href="/favicon.ico" />
+<!-- <link rel="stylesheet" type="text/css" href="/css/bootstrap.min.css"> -->
+<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css" integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z" crossorigin="anonymous">
+<link rel="stylesheet" type="text/css" href="/css/style.css" />
+<link rel="stylesheet" type="text/css" href="/css/neutra/stylesheet.css" />
+<link rel="stylesheet" type="text/css" href="/css/jquery-editable-select.min.css" />
+
 <script src="/js/jquery-3.2.1.min.js"></script>
 <script src="/js/jquery-ui.js"></script>
-<script src="/js/bootstrap.min.js"></script>
+<!-- <script src="/js/bootstrap.min.js"></script> -->
+<script src="https://code.jquery.com/jquery-3.5.1.min.js"></script>
```

Comment:
If you have time please also download jquery 3.5.1 replace the local 3.2.1 on line 20